dspace-spring-rest/src/test/java/org/dspace/app/rest/MetadatafieldRestRepositoryIT.java

```diff
@@ -124,4 +163,190 @@ public void findByNullSchema() throws Exception {
+        metadataFieldRest.setQualifier(QUALIFIER_UPDATED);
+        metadataFieldRest.setScopeNote(SCOPE_NOTE_UPDATED);
+
+        try {
+            getClient(getAuthToken(admin.getEmail(), password))
+                    .perform(put("/api/core/metadatafields/" + metadataField.getID())
+                            .content(new ObjectMapper().writeValueAsBytes(metadataFieldRest))
+                            .contentType(contentType))
+                    .andExpect(status().isOk());
+
+            metadataField = metadataFieldService.find(context, metadataField.getID());
+
+            assertEquals(ELEMENT_UPDATED, metadataField.getElement());
+            assertEquals(QUALIFIER_UPDATED, metadataField.getQualifier());
+            assertEquals(SCOPE_NOTE_UPDATED, metadataField.getScopeNote());
+        } finally {
+            deleteMetadataFieldIfExists(metadataField);
+        }
+    }
+
+    @Test
+    public void updateUnauthorized() throws Exception {
+
+        MetadataField metadataField = createMetadataField();
+
+        MetadataFieldRest metadataFieldRest = new MetadataFieldRest();
+        metadataFieldRest.setId(metadataField.getID());
+        metadataFieldRest.setElement(ELEMENT_UPDATED);
+        metadataFieldRest.setQualifier(QUALIFIER_UPDATED);
+        metadataFieldRest.setScopeNote(SCOPE_NOTE_UPDATED);
+
+        try {
+            getClient()
+                    .perform(put("/api/core/metadatafields/" + metadataField.getID())
+                            .content(new ObjectMapper().writeValueAsBytes(metadataFieldRest))
+                            .contentType(contentType))
+                    .andExpect(status().isUnauthorized());
+
+            metadataField = metadataFieldService.find(context, metadataField.getID());
+
+            assertEquals(ELEMENT, metadataField.getElement());
+            assertEquals(QUALIFIER, metadataField.getQualifier());
+            assertEquals(SCOPE_NOTE, metadataField.getScopeNote());
+        } finally {
+            deleteMetadataFieldIfExists(metadataField);
+        }
+    }
+
+    private MetadataField createMetadataField() throws AuthorizeException, SQLException, NonUniqueMetadataException {
+        context.turnOffAuthorisationSystem();
+        MetadataField metadataField = metadataFieldService.create(
+                context, metadataSchema, ELEMENT, QUALIFIER, SCOPE_NOTE
+        );
+        context.commit();
+        return metadataField;
+    }
+
+    private void deleteMetadataFieldIfExists() throws SQLException, AuthorizeException {
+
+        deleteMetadataFieldIfExists(metadataFieldService.findByElement(context, metadataSchema, ELEMENT, QUALIFIER));
+    }
+
+    private void deleteMetadataFieldIfExists(MetadataField metadataField) throws SQLException, AuthorizeException {
+        if (metadataField != null) {
+            context.turnOffAuthorisationSystem();
+            metadataFieldService.delete(context, metadataField);
+            context.commit();
+        }
```

Comment:
to be safe you should restore the authorisationSystem

Reply:
This code seems to be refactored and no longer exists.  I believe it was replaced with the `MetadataFieldBuilder` class.